app/utils/routeHelpers.js

```diff
@@ -14,10 +14,19 @@ export function newCollectionUrl(): string {
   return "/collections/new";
 }
 
-export function collectionUrl(collectionId: string, section: ?string): string {
-  const path = `/collections/${collectionId}`;
-  if (section) return `${path}/${section}`;
-  return path;
+export function collectionUrl(url: string, section: ?string): string {
+  if (section) return `${url}/${section}`;
+  return url;
+}
+
+export function updateCollectionUrl(oldUrl: string, newUrl: string): string {
```

Comment:
Wondering if this would be more maintainable as a string replacement, looking for `collections/${collection.id}` -> `collections.url` ?

Reply:
I am sorry I didn't get it. Isn't this what we are doing?

Reply:
I think that's what this is achieving but in a slightly more fragile way by splitting and recombining the parts. Could it not be?

```javascript
  return oldUrl.replace(`collections/${collection.id}`, collections.url);
```

Reply:
Do you suggest modify the `updateDocumentUrl`? This function is an exact copy of it, and just the name is different 😅

Reply:
Hah, maybe! My old code can always be improved... but maybe I'm missing something here, it's late

Reply:
I will update it if it doesn't break anything.